debian/pop-session.gsettings-override

```diff
@@ -54,14 +55,14 @@ clock-format = '12h'
 close = ['<Super>q']
 maximize = []
 minimize = []
-move-to-monitor-down = ['<Primary><Shift><Super>Down', '<Primary><Shift><Super>j']
-move-to-monitor-left = ['<Shift><Super>Left', '<Shift><Super>h']
-move-to-monitor-right = ['<Shift><Super>Right', '<Shift><Super>l']
-move-to-monitor-up = ['<Primary><Shift><Super>Up', '<Primary><Shift><Super>k']
-move-to-workspace-down = ['<Shift><Super>Down', '<Shift><Super>j']
+move-to-monitor-down = []
+move-to-monitor-left = []
+move-to-monitor-right = []
+move-to-monitor-up = []
+move-to-workspace-down = []
```

Comment:
Those two commits from Michael shouldn't be in this PR, right?

Reply:
Yeah, I accidentally opened against master_focal at first. Here's one for master_groovy: https://github.com/pop-os/session/pull/25